README.md

```diff
@@ -73,12 +73,12 @@ for details on how to define a space.
 
 ### Step 3: Provide Your Training and Test data
 
-Put your training and test data in `data/{train, validation}.md`
+Put your training and test data in `data/{training_data, test_data}.yml`
```

Comment:
```suggestion
Put your training and test data in `train_test_split/{training_data, test_data}.yml`
```

Here and elsewhere I'm suggesting to standardize on this, because it is the default output directory for a train-test-split.